miso-web/src/main/webapp/scripts/form_sample.js

```diff
@@ -94,6 +71,29 @@ FormTarget.sample = (function ($) {
               type: "text",
               maxLength: 255,
             },
+            {
```

Comment:
It was because in the bulk_sample.js file I moved the project section before the requisition field, so I thought I'd keep the order uniform. However, if there is copy-pasting happening, then I'll revert the change to prevent potential errors